docs/parameter_tables/HomSap/Zigzag_1S14.csv

```diff
@@ -1,5 +1,5 @@
-Population size,"1,431",Ancient pop. size
-Population size,"14,312",Recent pop. size
+Population size,"7,155",Ancient pop. size
```

Comment:
I think this [should be 7,156](https://github.com/popsim-consortium/stdpopsim/blob/main/stdpopsim/catalog/HomSap/demographic_models.py#L645)

Reply:
Ah, thanks. Was just looking at the previous values.